core/pfe-core/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # @patternfly/pfe-core
 
+## 2.1.0
+
+### Minor Changes
+
+- e45f5eb5a: `roving-tabindex-controller`: enabled controller to be used by aria-expanded elements
```

Comment:
@nikkimk since this is a minor change, can you add some prose and a snippet to this changeset, explaining how to hook into the new functionality?